### PR TITLE
update icon: appwrite (original, original-wordmark, plain, plain-wordmark)

### LINKED ISSUE #2181: [ICON REQUEST]: Appwrite
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

The icon is already in the project but it is outdated...

### Links and sources

- Appwrite's Official Website: https://appwrite.io/
- Appwrite's Official Assets Page: https://appwrite.io/assets
  - Original: https://appwrite.io/assets/logomark/logo.svg
  - White Text Wordmark (Original as we can see on their website): https://appwrite.io/assets/logotype/black.svg
  - Black Text Wordmark: https://appwrite.io/assets/logotype/white.svg

#### appwrite-original
</img>

#### appwrite-original-wordmark
</img>

#### appwrite-dark-wordmark
</img>

#### appwrite-plain
maybe shoud be removed?

#### appwrite-plain-wordmark
maybe shoud be removed?

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Yashh56**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #2181**

## Link to prove your SVG is correct and up-to-date.

- [Website](https://appwrite.io/)

The color `#fd366e` was selected for the font SVGs as it's the main color in the original logo.

### CODE REVIEW

**Comment by Yashh56** [comment]
**Hey Maintainer Please Review this PR**

File: icons/appwrite/appwrite-original.svg
Comment (Snailedlt) [inline icons/appwrite/appwrite-original.svg:1]: Looks like this icon has been distorted.

File: icons/appwrite/appwrite-plain.svg
Comment (Snailedlt) [inline icons/appwrite/appwrite-plain.svg:1]: Same here :)

**Review by Snailedlt — DISMISSED** [review]
Thank you for the contribution!

The wordmark versions look very good!

The non-wordmark versions are distorted. Please make sure to keep the aspect ratios the same when resizing the icons, so that they don't become distorted.

**Comment by Yashh56** [comment]
**Thanks @Snailedlt  for reviewing this PR Can you tell me how to resolve this issue ?**

**Comment by Snailedlt** [comment]
> **Thanks @Snailedlt  for reviewing this PR Can you tell me how to resolve this issue ?**

@Yashh56 I think the easiest way is downloading the official icons again and making sure that the aspect ratio is maintained when you resize it :)

If you're using inkscape you can follow this guide when resizing: https://github.com/devicons/devicon/issues/1306#issuecomment-1296051881

**Comment by Yashh56** [comment]
Hey @Snailedlt are these changes fine?

**Comment by Yashh56** [comment]
**Hey Maintainer Please Review this PR**

**Review by weh — DISMISSED** [review]
It looks like the changes requested by @Snailedlt are still not resolved.
The `original` and `plain` version are still squished, and not round anymore.

squished

**Comment by weh** [comment]
...also `original` and `plain` could be aliased since it is only a single color (?)

**Comment by weh** [comment]
And i forgot also that the original and plain should not have that much spacing around the icon. 
So to summarize again:

- Fix the squished aspect ratio
- Remove the spacing around the Icon
- Remove the `plain-wordmark` / `original-wordmark` alias

```json
        "aliases": [
            {
                "base": "plain",
                "alias": "original"
            }
        ]
```

**Comment by Snailedlt** [comment]
Refer to guides under [this issue](https://github.com/devicons/devicon/issues/1306) as well as the [wiki](https://github.com/devicons/devicon/wiki/Overview-on-Submitting-Icons) if you're unsure how to fix the issues :)

**Comment by Yashh56** [comment]
**how can i squished aspect ratio ?**

**Comment by Yashh56** [comment]
**I used Resize to resize the aspect ratio**

**Comment by Snailedlt** [comment]
@Yashh56 If you're using inkscape, make sure you check the box to scale proportionally
image
Otherwise the svg will get stretched to fit the viewbox.

Here's two different guides to resize to a square. Both can be found as comments in [the issue I linked to earlier](https://github.com/devicons/devicon/issues/1306):
- https://github.com/devicons/devicon/pull/1938#discussion_r1378205483
- https://github.com/devicons/devicon/pull/1465#issuecomment-1287950577

**Comment by Yashh56** [comment]
**Hey Maintainer Please review the current changes. **

**Comment by Yashh56** [comment]
**Hey @Snailedlt, Can you review my changes :)**

**Review by canaleal — APPROVED** [review]
Ive optimized the svgs using this [tutorial](https://github.com/devicons/devicon/wiki/Optimizing-icons) and made sure there were no artifacts in icomoon. 

The eps file can be removed as it's not needed anymore and because the original icon is simple enough to be used as the plain icon, the plain svg was removed and replaced with an alias. 

image

**Pull request merged**